Scripts/RemediationScripts/Remediate-SetClusterProtectionLevelForServiceFabric.ps1

```diff
@@ -645,10 +824,11 @@ function Set-ClusterProtectionLeveltoPreviousValueforServiceFabric
             if($userInput -ne "Y")
             {
                 Write-Host "Cluster Protection level will not be rolled back on Service Fabric(s) mentioned in the file. Exiting..." -ForegroundColor $([Constants]::MessageType.Warning)
-                break
+                 -ForegroundColor $([Constants]::MessageType.Warning)
+                return
             }
             Write-Host "Cluster Protection level will be rolled back on Service Fabric(s) mentioned in the file." -ForegroundColor $([Constants]::MessageType.Update)
```

Comment:
Single dash line below this

Reply:
added double dash line as mentioned in the above comments

Reply:
done